File: html/script/array1.js
var ar1=new Array(11,22,33,44,55)
alert(ar1)
alert("array1 length is "+ar1.length)

var city=["delhi","chennai","mumbai","nagpur","madurai"];
alert(city)

console.log(city[0])
console.log(city[2])
console.log(city[1])

document.write("using for loop ")

for(var a=0;a<city.length;a++)
   document.write(city[a]+"<br>");
   
document.write("using for in/each   loop ")

for(var a in ar1)// for in/each return index 
   document.write(ar1[a]+"<br>");
   
document.write("using  of  loop ")

for(var a of ar1)// for in/each return index 
   document.write(a+"<br>");
   
   

File: corejava/src/Array1.java

public class Array1 {

	public static void main(String[] args) {
		
		
		int ar1[]= {11,22,33,44,55};
		
		int ar2[]=new int[5];
		
		System.out.println(ar1[0]);
		System.out.println(ar1[1]);
		System.out.println(ar1[2]);
		System.out.println(ar1[3]);
		
		//System.out.println(ar1[5]);
		
		System.out.println(ar1.length);
		
		
		System.out.println(ar2[0]);
		System.out.println(ar2[1]);
		System.out.println(ar2[2]);
		System.out.println(ar2[3]);
		
		for (int i = 0; i < ar1.length; i++) {
			System.out.print(ar1[i]+",  ");
		}
		
		System.out.println("using for each loop ");
		
		for (int i : ar1) {
			System.out.print(i+", ");
		}
		
	}
}
